apps/web/src/services/dashboards/widgets/cost-widgets/aws-cloud-front-cost/AWSCloudFrontCostWidget.vue

```diff
@@ -250,28 +261,28 @@ const drawChart = (chartData) => {
         };
         const series = chartHelper.createXYColumnSeries(chart, seriesSettings);
         chart.series.push(series);
+        series.data.setAll(cloneDeep(chartData));
+
         const tooltip = chartHelper.createTooltip();
         tooltip.label.adapters.add('text', (text, target) => {
-            // let _text = `[${gray[700]}]{valueX}[/]`;
             let _text = `[${gray[700]}]${target.dataItem?.dataContext?.[groupBy]}[/]`;
             chart.series.each((s) => {
                 const fieldName = s.get('valueYField') || s.get('valueXField') || '' as UsageType;
                 let value = target.dataItem?.dataContext?.[fieldName];
+                const unit = (target.dataItem?.dataContext as ChartData)?.usage_unit ?? '';
                 if (value === undefined) value = '--';
-                if (fieldName === 'data-transfer.out') {
+                if (typeof value === 'number') {
                     if (selectedSelectorType.value === 'cost') {
-                        if (state.currency) value = currencyMoneyFormatter(value, widgetState.currency);
+                        if (widgetState.currency) value = currencyMoneyFormatter(value, widgetState.currency);
                     } else {
-                        value = bytes.parse(`${value}${USAGE_SOURCE_UNIT}`);
-                        value = byteFormatter(value);
+                        value = `${numberFormatter(value)}${unit ? ` (${unit})` : ''}`;
```

Comment:
Question: If there is no `usage_unit`, the table in costAnalysis does not show the value at all, but does the widget show the value even if there is no `usage_unit`?